setup.py

```diff
@@ -76,7 +76,7 @@ def read(fname):
     extras_require={
         'test': tests_require,
         's3': aws_deps,
-        'gcs': gcp_deps,
+        'gcs': gcs_deps,
```

Comment:
You also need to change the definition of gcp_deps to dcs_deps above ;)

Reply:
Please run a grep on `gcp` in the entire codebase + docs. Let's fix this once and for all! Thanks.